.freeCodeCamp/tooling/seed.js

```diff
@@ -17,23 +15,19 @@ const execute = promisify(exec);
 
 export async function seedLesson(ws, project) {
   // TODO: Use ws to display loader whilst seeding
-  const lessonNumber = project.currentLesson;
+  const { currentLesson: lessonNumber, dashedName: projectPath } = project;
```

Comment:
Nitpick: I would prefer these to remain named as is. Destructuring into different names just makes searching the codebase more annoying.